litecommands-annotations/test/dev/rollczi/litecommands/annotations/async/AsyncArgumentsInitializeTest.java

```diff
@@ -50,7 +50,6 @@ void test() {
         List<CompletableFuture<AssertExecute>> futures = new ArrayList<>();
         for (int i = 0; i < 50; i++) {
             TestPlatform platform = LiteCommandsTestFactory.startPlatform(config -> config
-                .scheduler(new SchedulerExecutorPoolImpl("lite-commands", 10))
```

Comment:
Please revert it, this unit test is used to check a case with the specific scheduler.